addons/website_forum/static/src/js/website_forum.js

```diff
@@ -465,11 +494,13 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
     _onFavoriteQuestionClick: function (ev) {
         ev.preventDefault();
         var $link = $(ev.currentTarget);
+        var $link_icon = $link.find('.fa');
```

Comment:
same here (and you can sue const and not var, even if we use jquery / var in this file we can try to clean that)